MapleServer2/Types/Inventory.cs

```diff
@@ -323,10 +323,52 @@ public void MoveItem(GameSession session, long uid, short dstSlot)
         session.Send(ItemInventoryPacket.Move(dstUid, srcSlot, uid, dstSlot));
     }
 
+    public bool HasItem(long uid)
```

Comment:
Can we use expression body for all these functions?

Reply:
Sure can!